app/views/search/_granule_list.html.erb

```diff
@@ -66,33 +62,44 @@
             <p class="granule-sort">Sort by: <select id="granule-sort" data-bind="options: granules().query.sortOptions, optionsText: 'name', optionsValue: 'value', value: granules().query.sortKey"></select></p>
           <!--/ko-->
 
+          <!-- ko if: granules().excludedGranulesList().length > 0 -->
+          <p class="undo">Granule excluded. <a href="#" data-bind="click: undoExcludeGranule">Undo</a></p>
           <!-- /ko -->
 
+        <!-- /ko -->
         </div>
         <div id="granules-scroll" class="master-overlay-content panel-list" data-bind="event: {scroll: scrolled}">
           <ul class="panel-list-list" data-bind="foreach: granules().results()">
 
             <li class="panel-list-item" data-bind="css: {'panel-list-item-with-thumb': browse_flag === 'true' || browse_flag === true, 'panel-list-focused': $parent.focused() === $data, 'panel-list-selected': $parent.isStickied($data)}, event: { mouseenter: $parent.onGranuleMouseover, mouseleave: $parent.onGranuleMouseout, click: $parent.toggleStickyFocus}">
+              <h3 data-bind="attr: {id: 'granule-list-item-' + $index()}, text: displayName()"></h3>
 
               <!-- ko if: browse_flag === 'true' || browse_flag === true -->
                 <a class="panel-list-thumbnail-container" data-bind="attr: {href: edsc_full_browse_url()}" target="_blank" title="View full image">
                   <img class="panel-list-thumbnail" data-bind="event: {error: onThumbError}, attr: {src: edsc_browse_url(<%= Rails.configuration.thumbnail_width %>)}" alt="Browse Image">
                 </a>
-                <!-- /ko -->
-                <!-- ko if: $parent.canRemoveGranules() -->
-                <a class="panel-list-remove panel-list-when-selected" data-bind="click: $parent.removeGranule" href="#" title="Exclude this granule"><i class="fa fa-times-circle"></i></a>
-                <!-- /ko -->
+              <!-- /ko -->
 
-              <h3 data-bind="attr: {id: 'granule-list-item-' + $index()}, text: displayName()"></h3>
-              <span class="granule-temporal-range" data-bind="text: getTemporal()"></span>
+              <div class="granule-temporal-range" data-bind="html: getTemporal()"></div>
 
               <div class="panel-list-item-actions granule-item-actions">
                 <a href="#" class="button icon-button master-overlay-forward" data-master-overlay-forward="Granules" data-bind="click: $parent.showGranuleDetails" title="View granule details">
                   <i class="fa fa-info-circle"></i>
                 </a>
-                <a href="#" class="button icon-button" title="Download single granule data" data-bind="click: $root.ui.projectList.loginAndDownloadGranule.bind($data, $parent.collection)">
+                <a class="button icon-button" title="Download single granule data" target="_blank" data-bind="visible: !$parent.collection.cwic() && online_access_flag, attr: {href: download_now_url()}">
```

Comment:
The more I think about it the less I want to include it on this PR. Disregard for now.